ccloud/ccloud-generate-cp-configs.sh

```diff
@@ -98,3 +133,259 @@ while read -r line
+props.put(StreamsConfig.PRODUCER_PREFIX + ProducerConfig.INTERCEPTOR_CLASSES_CONFIG + StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, confluent.monitoring.interceptor.bootstrap.servers, "$BOOTSTRAP_SERVERS");
+props.put(StreamsConfig.PRODUCER_PREFIX + ProducerConfig.INTERCEPTOR_CLASSES_CONFIG + StreamsConfig.SECURITY_PROTOCOL_CONFIG, "SASL_SSL");
+props.put(StreamsConfig.PRODUCER_PREFIX + ProducerConfig.INTERCEPTOR_CLASSES_CONFIG + SaslConfigs.SASL_MECHANISM, "PLAIN");
+props.put(StreamsConfig.PRODUCER_PREFIX + ProducerConfig.INTERCEPTOR_CLASSES_CONFIG + SaslConfigs.SASL_JAAS_CONFIG, "$SASL_JAAS_CONFIG");
+props.put(StreamsConfig.CONSUMER_PREFIX + ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG, "io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor");
+props.put(StreamsConfig.CONSUMER_PREFIX + ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG + StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, confluent.monitoring.interceptor.bootstrap.servers, "$BOOTSTRAP_SERVERS");
+props.put(StreamsConfig.CONSUMER_PREFIX + ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG + StreamsConfig.SECURITY_PROTOCOL_CONFIG, "SASL_SSL");
+props.put(StreamsConfig.CONSUMER_PREFIX + ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG + SaslConfigs.SASL_MECHANISM, "PLAIN");
+props.put(StreamsConfig.CONSUMER_PREFIX + ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG + SaslConfigs.SASL_JAAS_CONFIG, "$SASL_JAAS_CONFIG");
+
+// .... additional configuration settings
+EOF
+
+# Python
+PYTHON_CONFIG=$DEST/python.delta
+echo "$PYTHON_CONFIG"
+
+cat <<EOF >> $PYTHON_CONFIG
+from confluent_kafka import Producer, Consumer, KafkaError
+
+producer = Producer({
+           'bootstrap.servers': '$BOOTSTRAP_SERVERS',
+           'broker.version.fallback': '0.10.0.0',
+           'api.version.fallback.ms': 0,
+           'sasl.mechanisms': 'PLAIN',
+           'security.protocol': 'SASL_SSL',
+           'sasl.username': '$CLOUD_KEY',
+           'sasl.password': '$CLOUD_SECRET',
+           'plugin.library.paths': 'monitoring-interceptor',
+           // .... additional configuration settings
+})
+
+consumer = Consumer({
+           'bootstrap.servers': '$BOOTSTRAP_SERVERS',
+           'broker.version.fallback': '0.10.0.0',
+           'api.version.fallback.ms': 0,
+           'sasl.mechanisms': 'PLAIN',
+           'security.protocol': 'SASL_SSL',
+           'sasl.username': '$CLOUD_KEY',
+           'sasl.password': '$CLOUD_SECRET',
+           'plugin.library.paths': 'monitoring-interceptor',
+           // .... additional configuration settings
+})
+EOF
+
+# .NET 
+DOTNET_CONFIG=$DEST/dotnet.delta
+echo "$DOTNET_CONFIG"
+
+cat <<EOF >> $DOTNET_CONFIG
+using Confluent.Kafka;
+
+var producerConfig = new Dictionary<string, object>
+{
+    { "bootstrap.servers", "$BOOTSTRAP_SERVERS" },
+    { "broker.version.fallback", "0.10.0.0" },
+    { "api.version.fallback.ms", 0 },
+    { "sasl.mechanisms", "PLAIN" },
+    { "security.protocol", "SASL_SSL" },
+    { "ssl.ca.location", "/usr/local/etc/openssl/cert.pem" }, // linux, osx
```

Comment:
I don't believe this is a standard install path on Linux.
Also, why is this needed for the .NET client but not Python and Go? They should behave identically in this regard

Reply:
> I don't believe this is a standard install path on Linux.

This is verbatim taken from https://github.com/confluentinc/confluent-kafka-dotnet/blob/master/examples/ConfluentCloud/Program.cs#L57 . I actually don't think we need to iterate on all OS types for the example, users should change the path for their system.  So I'm ok with no change here.

> Also, why is this needed for the .NET client but not Python and Go? They should behave identically in this regard

I don't know if {{ssl.ca.location}} omission is accidental or intentional from Python and Go examples for CCloud.

* Python: https://github.com/confluentinc/confluent-kafka-python/blob/master/examples/confluent_cloud.py
* Go: https://github.com/confluentinc/confluent-kafka-go/blob/master/examples/confluent_cloud_example/confluent_cloud_example.go

cc: @mhowlett for additional insight